docker/Dockerfile.firedrake

```diff
@@ -7,6 +7,10 @@ RUN pip install --verbose \
         --extra-index-url https://download.pytorch.org/whl/cpu \
         jax ngsPETSc torch vtk
 
+# Set PYTHONPATH so netgen can be found
+# (see https://github.com/NGSolve/netgen/issues/213)
+ENV PYTHONPATH=/usr/local/lib/python3.12/site-packages
```

Comment:
Usually one adds to paths even if they are currently empty. E.g. someone might in future need to add to PYTHONPATH in firedrake-vanilla-default, which this would then break.

Reply:
```suggestion
ENV PYTHONPATH=/usr/local/lib/python3.12/site-packages:$PYTHONPATH
```